### PR TITLE
Feature/http2 by default

### PR DESCRIPTION
**Pull request by bellackn**

### CODE REVIEW

File: changes.xml [hunk 1]
```diff
@@ -24,6 +24,9 @@
   <body>
 
     <release version="1.6.0" date="not released">
+      <action type="add" dev="nbellack">
+        Role aem-dispatcher: Enable HTTP/2 by default.
```
Comment (stefanseifert) [inline changes.xml]: make clear that HTTP/2 support was added with this release (an it's enabled by default).
as it's written currently it sounds that support was available already and is now only enabled by default.

File: conga-aem-definitions/src/main/templates/aem-dispatcher/author/vhost_author.ssl.conf.hbs [hunk 2]
```diff
@@ -7,6 +7,11 @@
 # Virtual host for author (HTTPS)
 <VirtualHost {{httpd.serverListenAddressSsl}}:{{httpd.serverPortSsl}}>
 
+{{~#if httpd.http2.enabled}}
+# Enable HTTP/2
```
Comment (stefanseifert) [inline conga-aem-definitions/src/main/templates/aem-dispatcher/author/vhost_author.ssl.conf.hbs]: i think this block should be moved to vhost_author.partials.hbs
and put further down in this file, e.g. after the block "serverAliasNames"

File: conga-aem-definitions/src/main/templates/aem-dispatcher/publish/vhost_publish_tenant.ssl.conf.hbs [hunk 3]
```diff
@@ -7,6 +7,11 @@
 # Virtual host for publish site of tenant {{tenant}} (HTTPS)
 <VirtualHost {{httpd.serverListenAddressSsl}}:{{httpd.serverPortSsl}}>
 
+{{~#if httpd.http2.enabled}}
```
Comment (stefanseifert) [inline conga-aem-definitions/src/main/templates/aem-dispatcher/publish/vhost_publish_tenant.ssl.conf.hbs]: i think this block should be moved to vhost_publish_tenant.partials.hbs
and put further down in this file, e.g. after the block "serverAliasNames"

**Comment by tobias-richter** [comment]
@stefanseifert you can do a "Squash and merge" via the GitHub UI so from my perspective no squashed commit needed.

File: changes.xml (hunk 1 above)
Comment (bellackn) [reply changes.xml]: Changed in latest commit

File: conga-aem-definitions/src/main/templates/aem-dispatcher/author/vhost_author.ssl.conf.hbs (hunk 2 above)
Comment (bellackn) [reply conga-aem-definitions/src/main/templates/aem-dispatcher/author/vhost_author.ssl.conf.hbs]: Changed in latest commit

File: conga-aem-definitions/src/main/templates/aem-dispatcher/publish/vhost_publish_tenant.ssl.conf.hbs (hunk 3 above)
Comment (bellackn) [reply conga-aem-definitions/src/main/templates/aem-dispatcher/publish/vhost_publish_tenant.ssl.conf.hbs]: Changed in latest commit

**Comment by stefanseifert** [comment]
> 
> 
> @stefanseifert you can do a "Squash and merge" via the GitHub UI so from my perspective no squashed commit needed.

i known, but often i've to do some minor last touches on the PRs and i want to do this in the same branch before merging it in develop, but i do not want to squash this together with the other commits to retain the original author information of the contributor.

File: conga-aem-definitions/src/main/templates/aem-dispatcher/author/vhost_author.ssl.conf.hbs [hunk 4]
```diff
@@ -21,6 +21,14 @@ ServerAlias {{this}}
 {{~/if}}
 {{/partial}}
 
+{{~#partial "http2"}}
```
Comment (stefanseifert) [inline conga-aem-definitions/src/main/templates/aem-dispatcher/author/vhost_author.ssl.conf.hbs]: this is redundant, please remove.

File: conga-aem-definitions/src/main/templates/aem-dispatcher/publish/vhost_publish_tenant.ssl.conf.hbs [hunk 5]
```diff
@@ -21,6 +21,14 @@ ServerAlias {{this}}
 {{~/if}}
 {{/partial}}
 
+{{~#partial "http2"}}
```
Comment (stefanseifert) [inline conga-aem-definitions/src/main/templates/aem-dispatcher/publish/vhost_publish_tenant.ssl.conf.hbs]: this is redundant, please remove.

**Comment by stefanseifert** [comment]
thanks, it's merged

**Pull request closed without merging**